Herebyfile.mjs

```diff
@@ -610,7 +611,7 @@ export const watchLocal = task({
     dependencies: [localize, watchTsc, watchTsserver, watchServices, watchLssl, watchOtherOutputs, dts, watchSrc],
 });
 
-const runtestsDeps = [tests, generateLibs].concat(cmdLineOptions.typecheck ? [dts, buildSrc] : []);
+const runtestsDeps = [tests, generateLibs].concat(cmdLineOptions.typecheck ? [dts] : []);

```

Comment:
This is the one odd case; when typechecking is disabled, I skip the dts task. This is how we can go fast for quick F5s.

Potentially, this should be its own flag for tasks.json instead.